tests/test_pyglmnet.py

```diff
@@ -122,16 +122,40 @@ def test_group_lasso():
     beta = np.random.normal(0.0, 1.0, n_features)
     beta[groups == 2] = 0.
 
-    # create an instance of the GLM class
-    glm_group = GLM(distr='softplus', alpha=1.)
+    lams = [0.5, 0.3237394, 0.2096144, 0.13572088, 0.08787639,
+            0.0568981, 0.03684031, 0.02385332, 0.01544452, 0.01]
```

Comment:
I understand.
I will choice 0.2 as one `lam` value.
if `lam==0.2` in that case, I made sure that some groups to be zero, others not to be zero.
So I think it will be a good test case.